### DISCUSSION TITLE
Meeting: October 8th, 2024

### DISCUSSION THREAD

**Discussion by earth2marsh** [OAI/sig-moonwalk#156] [Announcements]
## Weekly meetings happen on Tuesdays

**Time:** 5pm GMT / 9am Pacific
**Zoom:** [https://zoom.us/j/975841675](https://zoom.us/j/975841675?pwd=SUh4MjRLaEFKNlI3RElpWTdhRDVVUT09), dial-in passcode: 763054

### Accessibility & Etiquette

* Participants must abide by our [Code-of-Conduct](https://github.com/OAI/OpenAPI-Specification/blob/main/CODE_OF_CONDUCT.md#code-of-conduct).
* Meetings are recorded for future reference, and for those who are not able to attend in-person.
* We invited challenges to language or behaviour that is harmful or not inclusive during this meeting.
* We look forward to your participation, but please consider these acts of etiquette:
  * Remain on mute when not speaking to prevent interruptions.
  * Blur your background to reduce visual distractions.
  * Use the Zoom meeting "Raise Hand" feature to notify the group when you wish to speak.

| Blur My Background | Raise Hand |
|-|-|
| <img width="323" alt="Screenshot of Zoom UI showing the 'Stop Video' and 'Blur My Background' control" src="https://github.com/OAI/OpenAPI-Specification/assets/7367/7e43dbbb-6529-46e6-8b04-4c1aa852d9dd"> | <img width="323" alt="Screenshot of Zoom UI showing the 'Reaction' and 'Raise Hand' control" src="https://github.com/OAI/OpenAPI-Specification/assets/7367/f991722f-4651-40aa-9bc4-7e9a2a165a6a"> |

### Agenda Structure

| Topic | Owner | Decision/NextStep |
|-------|---------|---------|  
| | | |
Intros and governance meta-topics (5 mins) | SIG members | -- |
General discussion topics (list below) | SIG members | -- |
Wrap-up and seed the next agenda (5 mins) | SIG members | -- |

Whether attending or not, **anyone can comment on this issue prior to the meeting to suggest topics or to add comments on planned topics or proposals**.

**Comment by earth2marsh** [comment]
Continuing from [last week's discussion](https://github.com/OAI/sig-moonwalk/discussions/155#discussioncomment-10742106), let's keep iterating through the other colors in the idealized architecture diagram from @handrews 's [blog post](https://modern-json-schema.com/analyzing-the-openapi-tooling-ecosystem), starting with blue!

**Comment by earth2marsh** [comment]
[Video](https://zoom.us/rec/share/Vedb3iFEl9Imz_CjQSd2lFloqAa3PJ9KOmx8d03lhlU5BmsYgEBoDa1YXbvghoBg._QKdnUznN7GxiQub) from yesterday's discussion, where we covered: 

- The possibility of flipping the round-trip ("tram lines") of a request/response so the consumer is on the right side, furthers from the OAD Tools section.
- Considered how the "tram lines" affect the diagram today
- Mused about the lumping/splitting/naming of the current 4 "API Tools" groupings
- Briefly discussed developing an ontology (Gary has that ball)

**Reply by handrews** [reply]
> The possibility of flipping the round-trip ("tram lines") of a request/response so the consumer is on the right side, furthers from the OAD Tools section.

It would be a lot easier to just move the OAD Tools to the right and the API Tools to the left.  Otherwise you'd need to flip the whole API Tools section to reverse the tram lines, and that would be a massive pain to do.  Although, really, the diagram was intended as a conversation tool and not as final accurate documentation on the ecosystem.

> Mused about the lumping/splitting/naming of the current 4 "API Tools" groupings

There are definitely other ways they could be grouped or subdivided.  I think this would be best done by figuring out what makes tools most easy to build than by moving the boxes based on the labels (some of which are intentionally vague).
